packages/veto-frontend/src/components/molecules/SideMenu/SideMenu.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent } from 'react'
+import { Flex } from '@veriphi/veto-ui'
+import { MainLayoutLabel, MainLayoutRoute } from '../../../layouts/MainLayout/MainLayout'
```

Comment:
We have absolute imports working in `veto-frontend` package with `baseUrl='src'` so you should be able to `import from 'layouts/MainLayout`